src/Logging.jl

```diff
@@ -48,9 +48,10 @@ type SysLog
     facility::LogFacility
     machine::AbstractString
     user::AbstractString
+    maxsize::UInt16
 
-    SysLog(host::AbstractString, port::Int) = new(UDPSocket(), getaddrinfo(host), (@compat UInt16(port)), LOG_USER, gethostname(), Base.source_path()==nothing ? "" : basename(Base.source_path()))
-    SysLog(host::AbstractString, port::Int, facility::LogFacility, machine::AbstractString, user::AbstractString) = new(UDPSocket(), getaddrinfo(host), (@compat UInt16(port)), facility, machine, user)
+    SysLog(host::AbstractString, port::Int) = new(UDPSocket(), getaddrinfo(host), (@compat UInt16(port)), LOG_USER, gethostname(), Base.source_path()==nothing ? "" : basename(Base.source_path()), (@compat UInt16(1024)))
```

Comment:
These lines are just getting too long aren't they.  maxsize isn't available here it will just use the "default" of 1024.